packages/components/src/range-control/README.md

```diff
@@ -193,6 +229,86 @@ The maximum value accepted. If higher values are inserted onChange will not be c
 - Required: No
 - Platform: Web | Mobile
 
+#### renderTooltipContent
+
+A way to customize the rendered UI of the value. Example:
+
+```jsx
+const customTooltipContent = value => `${value}%`
+
+const MyRangeControl() {
+	return (<RangeControl renderTooltipContent={ customTooltipContent } />)
+}
+```
+
+- Type: `Function`
+- Required: No
+- Platform: Web
+
+#### showTooltip
+
+Forcing the Tooltip UI to show or hide.
+
+- Type: `Boolean`
+- Required: No
+- Platform: Web
+
+#### step
+
+The stepping interval between `min` and `max` values. Step is used both for user interface and validation purposes.
+
+- Type: `Number`
+- Required: No
+- Platform: Web
+
+#### tooltipPosition
+
+Determines the render position of the Tooltip. Either `auto` (default), `top`, or `bottom`.
+
+- Type: `String`
+- Required: No
+- Platform: Web
+
+#### tooltipTimeout
+
+Determines the `timeout` (ms) to hide the Tooltip based on mouse movements.
+
+- Type: `Number`
+- Required: No
+- Platform: Web
+
+#### tooltipZIndex
+
+Determines the CSS `z-index` property of the Tooltip.
+
+- Type: `Number`
+- Required: No
+- Platform: Web
```

Comment:
Do we really need all these customization options: `tooltiipPosition`, `tooltipTimetout` and `tooltipZIndex`? I think generally speaking for regular design systems, they tend to be very liberal to add props and variations but in the context of @wordpress/components we prefer the opposite because the backward compatibility strategy is stricter than what npm packages do in general.

Maybe we can just provide sane defaults and introduce props if we really get important use-cases for them?

Reply:
> Maybe we can just provide sane defaults and introduce props if we really get important use-cases for them?

Can do. They already have default values. I'll remove the public facing APIs for these.